src/lua/api-gateway/validation/oauth2/oauthTokenValidator.lua

```diff
@@ -166,15 +166,8 @@ function _M:getTokenFromCache(cacheLookupKey)
     return nil;
 end
 
--- imsAuth will validate the service token passed in "Authorization" header --
-function _M:validate_ims_token()
+function _M:validateOAuthToken(oauth_token, validation_config)
```

Comment:
Another note: I initially wanted to add the `validation_config` object as a instance member, as I thought this would make more sense. But I'm not really proficient in Lua OO, so I followed an alternate route. WDYT?